setup.py

```diff
@@ -15,7 +15,7 @@
     # on changes in these frameworks
     "fastapi": (
         [
-            "respx==0.16.3",
+            "respx==0.19.2",
```

Comment:
This `respx` had some dependencies that weren't matching with the current constraints and hence the tests were failing.

@rishabhpoddar respx is used only to mock `httpx` requests in the tests. It's not used anywhere else. So it's completely safe to upgrade it as long as tests pass and `make check-lint` doesn't throw error.

Reply:
okay